gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/StoreType.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.gaas.model;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum StoreType {
```

Comment:
Exactly what we wanted to move away from - hard coded values... See #79

Reply:
Do you want it interfaced so that Developer Users can implement and add their own GafferSpecBuilder jars - And those Users can hardcode their own Strings with their own custom store types?

Is that what you mean by more maintainable?

If so - we can make the API more extendable for Users to customise it by inserting their own custom Factory classes - a bit like the what can be done with custom UserFactory/spring-rest Docker image - and make it more of a thinner layer to interface with the Gaffer helm chart - we can even add an endpoint to allow Users to pass their own helm Values object in a request body for the gaffer-controller to deploy?

That way the Gaffer helm chart values change and you won't need to change the gaas-rest API implementation as much.

Reply:
See #99